docs/topics/native/native-binary-licenses.md

```diff
@@ -0,0 +1,76 @@
+[//]: # (title: License files for the Kotlin/Native binaries)
+
+Like many other open-source projects, Kotlin relies on third-party work. This means that the Kotlin project also
+includes the code developed not by JetBrains s.r.o. or the Kotlin programming language contributors. Sometimes it is not
+the exact code but derived work, like the code rewritten from C++ to Kotlin.
+
+> You can find licenses for the third-party work used in the Kotlin compiler in
+> our [GitHub repository](https://github.com/JetBrains/kotlin/tree/master/license/third_party).
+>
+{type="tip"}
+
+In particular, the Kotlin/Native compiler produces binaries that can include third-party code, data, or derived work.
+This means that the Kotlin/Native-compiled binaries are subject to the terms and conditions of the corresponding
+licenses.
+
+In practice, if you distribute a Kotlin/Native-compiled [final binary](https://kotlinlang.org/docs/multiplatform-build-native-binaries.html),
+you should always include necessary license files into your binary distribution. The files in a readable form should be
+accessible to users of your distribution.
+
+> You can find licenses for the third-party work used in Kotlin/Native in
+> our [GitHub repository](https://github.com/JetBrains/kotlin/tree/master/kotlin-native/licenses/third_party).
+>
+{type="tip"}
+
+Always include the following license files for the corresponding projects:
+
+<table>
+   <tr>
+      <th>Project</th>
+      <th>Files to be included</th>
+   </tr>
+   <tr>
+        <td><a href="https://kotlinlang.org/">Kotlin</a></td>
+        <td rowspan="3">
+         <list>
+            <li><a href="https://github.com/JetBrains/kotlin/blob/master/license/LICENSE.txt">Apache license 2.0</a></li>
+            <li><a href="https://github.com/JetBrains/kotlin/blob/master/kotlin-native/licenses/third_party/harmony_NOTICE.txt">Copyright notice</a></li>
```

Comment:
Better be more specific in the link title -- "Harmony copyright notice" or something. (I know the draft used the same wording, my bad).